src/main/java/uk/gov/hmcts/darts/task/service/impl/AutomatedTaskServiceImpl.java

```diff
@@ -312,78 +317,85 @@ private void addInboundToUnstructuredTaskRegistrar(ScheduledTaskRegistrar taskRe
      */
     private void addExternalDataStoreDeleterToTaskRegistrar(ScheduledTaskRegistrar taskRegistrar) {
         ExternalDataStoreDeleterAutomatedTask externalDataStoreDeleterAutomatedTask = new ExternalDataStoreDeleterAutomatedTask(
-            automatedTaskRepository,
-            lockProvider,
-            automatedTaskConfigurationProperties,
-            inboundDataStoreDeleter, unstructuredDataStoreDeleter, outboundDataStoreDeleter
+                automatedTaskRepository,
+                lockProvider,
+                automatedTaskConfigurationProperties,
+                inboundDataStoreDeleter, unstructuredDataStoreDeleter, outboundDataStoreDeleter
         );
-        externalDataStoreDeleterAutomatedTask.setLastCronExpression(getAutomatedTaskCronExpression(
-            externalDataStoreDeleterAutomatedTask));
+        externalDataStoreDeleterAutomatedTask.setLastCronExpression(getAutomatedTaskCronExpression(externalDataStoreDeleterAutomatedTask));
         Trigger trigger = createAutomatedTaskTrigger(externalDataStoreDeleterAutomatedTask);
         taskRegistrar.addTriggerTask(externalDataStoreDeleterAutomatedTask, trigger);
     }
 
     private void addCloseNonCompletedTranscriptionsAutomatedTaskToTaskRegistrar(ScheduledTaskRegistrar taskRegistrar) {
         CloseUnfinishedTranscriptionsAutomatedTask closeUnfinishedTranscriptionsAutomatedTask = new CloseUnfinishedTranscriptionsAutomatedTask(
-            automatedTaskRepository,
-            lockProvider,
-            automatedTaskConfigurationProperties,
-            transcriptionsApi
+                automatedTaskRepository,
+                lockProvider,
+                automatedTaskConfigurationProperties,
+                transcriptionsApi
         );
-        closeUnfinishedTranscriptionsAutomatedTask.setLastCronExpression(getAutomatedTaskCronExpression(
-            closeUnfinishedTranscriptionsAutomatedTask));
+        closeUnfinishedTranscriptionsAutomatedTask.setLastCronExpression(getAutomatedTaskCronExpression(closeUnfinishedTranscriptionsAutomatedTask));
         Trigger trigger = createAutomatedTaskTrigger(closeUnfinishedTranscriptionsAutomatedTask);
```

Comment:
indentation looks a bit weird here - but we seem to be having issues all over lately with formatting!